.circleci/config.yml

```diff
@@ -456,6 +462,53 @@ jobs:
       - run: bazel build src/... --build_tag_filters=-docs-package,-e2e --define=compile=aot
       - run: bazel test src/... --build_tag_filters=-docs-package,-e2e --test_tag_filters=-e2e --define=compile=aot
 
+  # ----------------------------------------------------------------------------
+  # Job that runs all Bazel tests against material-components-web#master.
+  # ----------------------------------------------------------------------------
+  mdc_snapshot_test_cronjob:
+    <<: *job_defaults
+    resource_class: xlarge
+    environment:
+      GCP_DECRYPT_TOKEN: *gcp_decrypt_token
+      MDC_REPO_URL: "https://github.com/material-components/material-components-web.git"
+      MDC_REPO_BRANCH: "master"
+      MDC_REPO_TMP_DIR: "/tmp/material-components-web"
+    steps:
+      - *checkout_code
+      - *restore_cache
+      - *setup_bazel_binary
+      - *setup_bazel_ci_config
+      - *setup_bazel_remote_execution
+      - *yarn_download
+      - *yarn_install
+
+      - run: git clone ${MDC_REPO_URL} --branch ${MDC_REPO_BRANCH} --depth 1 ${MDC_REPO_TMP_DIR}
+      - restore_cache:
+          keys:
+            - *mdc_deps_cache_key
+            - *mdc_deps_fallback_cache_key
+      - run:
+          name: "Installing dependencies for MDC repository"
+          # MDC repository does not use Yarn for node dependencies, so in order to respec the
+          # lock-file we need to use "npm" when installing dependencies.
+          command: cd ${MDC_REPO_TMP_DIR} && npm install
```

Comment:
@abhiomkar have you all considered switching to `yarn`? We've generally found that the locking mechanism works better (especially WRT remote caching and the bazel stuff we do)